runironic-api.sh

```diff
@@ -2,4 +2,13 @@
 
 . /bin/configure-ironic.sh
 
+# Wait for ironic to load all expected drivers
+# the DB query returns a string like ["fake-hardware", "idrac", "ipmi"]
+DB_DRIVERS=$(mysql --user=ironic --password=${MARIADB_PASSWORD} --protocol=tcp -r -s -e "use ironic; select drivers from conductors;" | jq '.|length')
```

Comment:
Not sure if jq pull in much dependencies but if its an issue we could 
mysql -N -u ironic -p${MARIADB_PASSWORD} -h 127.0.0.1 -e 'select drivers from conductors' ironic | grep -oE '"[^"]+"' | wc -l

Reply:
Ack, I don't mind which approach tbh, jq looks to only pull in one additional package beyond glibc:

```
$ sudo yum deplist jq | grep provider | grep -v i686 | sort | uniq
Repodata is over 2 weeks old. Install yum-cron? Or run: yum makecache fast
   provider: glibc.x86_64 2.17-292.el7
   provider: oniguruma.x86_64 5.9.5-3.el7
```

Reply:
Went with the grep and removed the jq dependency, thanks!